shanoir-ng-front/src/app/shared/components/entity/entity.abstract.service.ts

```diff
@@ -71,40 +70,43 @@ export abstract class EntityService<T extends Entity> implements OnDestroy {
             .toPromise();
     }
 
-    deleteWithConfirmDialog(name: string, entity: Entity): Promise<boolean> {
+    deleteWithConfirmDialog(name: string, entity: Entity, studyListStr?: string): Promise<boolean> {
+        let dialogTitle : string = 'Delete ' + name;
+        let dialogMsg : string = 'Are you sure you want to delete the ' + name
```

Comment:
@pierrehenri-dauvergne please change: "Are you sure you want to finally delete the "